guides/04_chatbots/01_creating-a-chatbot-fast.md

```diff
@@ -74,6 +74,7 @@ gr.ChatInterface(alternatingly_agree).launch()
 ## Streaming chatbots
 
 If in your chat function, you use `yield` to generate a sequence of responses, you'll end up with a streaming chatbot. It's that simple!
+Note that each yielded response replaces the previous one (and is not appended to the previously yielded content).
```

Comment:
I think we can rephrase the first paragraph instead of adding an additional one.

Reply:
Feedback addressed ;-)